src/wallet/walletutil.cpp

```diff
@@ -59,11 +60,16 @@ std::vector<fs::path> ListWalletDir()
     const fs::path wallet_dir = GetWalletDir();
     const size_t offset = wallet_dir.string().size() + 1;
     std::vector<fs::path> paths;
-    boost::system::error_code ec;
+    boost::system::error_code ec, eci;
```

Comment:
Nice catch. I can add commits to this PR fixing all other instances with missing error_code. (exists(), is_directory(), etc.)

For iterator() and increment() you have to use separate error_code because you have to handle them differently. Using single one it's hard to differentiate which one failed.

Reply:
Made PR #18189.